src/main/scala/gitbucket/core/service/IssueCreationService.scala

```diff
@@ -0,0 +1,75 @@
+package gitbucket.core.service
+
+import gitbucket.core.controller.Context
+import gitbucket.core.model.Issue
+import gitbucket.core.model.Profile.profile.simple.Session
+import gitbucket.core.service.RepositoryService.RepositoryInfo
+import gitbucket.core.util.Notifier
+import gitbucket.core.util.Implicits._
+
+/**
+  * Created by sk on 1/5 005.
+  */
+trait IssueCreationService {
```

Comment:
umm.. My purpose was to share createIssue with IssueController and ApiController.